openssl.conf

```diff
@@ -47,3 +48,4 @@ subjectKeyIdentifier = hash
 authorityKeyIdentifier = keyid,issuer:always
 keyUsage = critical, digitalSignature, keyEncipherment
 extendedKeyUsage = serverAuth
+subjectAltName=DNS:localhost
```

Comment:
My thoughts as to the most straightforward way to implement these
1) Can be done right now with no risk or harm to existing users or future APIs, by adding the above to the openssl configuration template.
    - This library would have to be modified so that the openssl config in `~/.config/devcert` is "upgraded", since as-is, it's created when the cert authority is set up and then never touched again.
   - I don't believe there's any harm in "updating" the openssl config as each signed certificate is generated, providing the serial number and database remain unchanged.

2) Can be accomplished by just providing instructions for editing `~/.config/devcert/openssl.conf` (and the equivalent for windows/linux) in README.md

3) Could be implemented by adding the ability to pass an optional options object to the devcert function, with devcert presenting a blocking Y/N confirmation prompt to the user, listing the domains to the console (thus making it more difficult for the user to be tricked into creating trusted certificates for `api.my-online-bank.com`)